src/backend/common/err_common.cpp

```diff
@@ -93,12 +93,13 @@ const string &DimensionError::getExpectedCondition() const { return expected; }
 
 int DimensionError::getArgIndex() const { return argIndex; }
 
-void print_error(const string &msg) {
+af_err set_error(const string &msg, af_err err) {
```

Comment:
something like `set_global_error_string` is more understandable and less misleading.